platform/ethereum/domain.go

```diff
@@ -32,10 +51,10 @@ func (p *Platform) Lookup(coins []uint64, name string) ([]blockatlas.Resolved, e
 	return result, nil
 }
 
-func (p *Platform) addressForCoin(resovler string, node []byte, coin uint64) (string, error) {
-	result, err := p.ens.Addr(resovler, node, coin)
+func (p *Platform) addressForCoin(resovler string, node []byte, coinType uint64) (string, error) {
```

Comment:
coinType 😭😩🤦‍♂️
Please, rename it to coinID because it is coinID

Reply:
ah..., I think this legacy code was written by me, we have enum `CoinType` in cpp
BTW, `coin` as package name is too generic, so I added alias

Reply:
coinID is nice, done